.make.versions

```diff
@@ -77,8 +77,8 @@ KFP_DOCKER_VERSION_v2=$(DPK_VERSION)
 ################## ################## ################## ################## ################## ##################
 # Begin versions that the repo depends on. 
 
-KFP_v2=2.7.0
-KFP_v1=1.8.22
+KFP_v2=2.2.0
```

Comment:
did you have any issues with kfp 2.7.0 ?

Reply:
[kfp/pipelines](https://github.com/kubeflow/pipelines) repo does not have tag 2.7.0